distribution/openhabhome/configurations/openhab_default.cfg

```diff
@@ -2464,3 +2464,8 @@ tcp:refreshinterval=250
 
 # Automatic sync time at openHAB startup (optional, defaults to false)
 #powermax:autoSyncTime=false
+
+################################ Megadevice Binding #######################################
+#
+# IP address of a Http port for megadevice defaut value is 8989
```

Comment:
This is not an IP address. Suggested text: "The http port to use for the megadevice binding. (optional, defaults to 8989)"